modules/apps/transfer/types/expected_keepers.go

```diff
@@ -39,6 +39,8 @@ type ChannelKeeper interface {
 	GetChannel(ctx sdk.Context, srcPort, srcChan string) (channel channeltypes.Channel, found bool)
 	GetNextSequenceSend(ctx sdk.Context, portID, channelID string) (uint64, bool)
 	GetAllChannelsWithPortPrefix(ctx sdk.Context, portPrefix string) []channeltypes.IdentifiedChannel
+
```

Comment:
[nit] can we axe this whitespace :sweat_smile:

Reply:
Done